content/.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1 @@
+This repository is auto-generated. Please open pull requests against https://github.com/prometheus/docs instead.
```

Comment:
Stronger language necessary? "You have to open pull requests against ... instead."?